pyproject.toml

```diff
@@ -0,0 +1,159 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "newa"
+dynamic = ["version"]
+description = "newa"
+readme = "README.md"
+# TODO: provide correct values
+# license = "MIT"
+# license-files = { paths = ["LICENSE"] }
+# requires-python = ">=3.9"
```

Comment:
@kkaarreell what Python versions do you expect to use? What would be the minimal one? 3.9 seems like a good one, it certainly works for tmt.

Reply:
I would like to preserve (eventually) the option to run tests using `tmt` which means we should use the same requirement.